### ISSUE TITLE
TypeError: Cannot destructure property `readFile` of 'undefined' or 'null'.

### ISSUE THREAD

**Issue by matheus-moreira** [commonshost/contact#2]
TypeError: Cannot destructure property `readFile` of 'undefined' or 'null'.
    at Object.<anonymous> (/usr/local/lib/node_modules/@commonshost/cli/src/options.js:3:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@commonshost/cli/src/commands/config.js:1:97)
    at Module._compile (module.js:652:30)

I'm getting this error with every command. What should I do?

**Comment by sebdeckers** [comment]
Which version of Node.js are you using?

```shell
$ node -v
v13.12.0
```

```shell
$ npm version
{
  npm: '6.14.4',
  ares: '1.16.0',
  brotli: '1.0.7',
  cldr: '36.1',
  icu: '66.1',
  llhttp: '2.0.4',
  modules: '79',
  napi: '6',
  nghttp2: '1.40.0',
  node: '13.12.0',
  openssl: '1.1.1e',
  tz: '2019c',
  unicode: '13.0',
  uv: '1.35.0',
  v8: '7.9.317.25-node.30',
  zlib: '1.2.11'
}
```

**Comment by matheus-moreira** [comment]
```
$ node -v
v8.10.0
```

```
$ npm version
{ npm: '6.14.4',
  ares: '1.14.0',
  cldr: '32.0.1',
  http_parser: '2.7.1',
  icu: '60.2',
  modules: '57',
  nghttp2: '1.30.0',
  node: '8.10.0',
  openssl: '1.0.2n',
  tz: '2017c',
  unicode: '10.0',
  uv: '1.18.0',
  v8: '6.2.414.50',
  zlib: '1.2.11' }
```

**Comment by sebdeckers** [comment]
TL;DR upgrade your Node.js to v10 or later.

It seems that Node.js version 10 or later is required for the CLI tool. A month ago I switched to using the Promises-based File System API which was not yet available in Node.js version 8.

FWIW the Node.js version you are running is also no longer supported by the Node.js project. You could try installing Node.js v12, or wait a week for v14 to become Current.

I personally use the odd version numbers. These do not receive LTS but instead always offer the latest features for Node.js development.

References:
- https://nodejs.org/api/fs.html#fs_filehandle_readfile_options
- https://gitlab.com/commonshost/cli/-/blob/master/src/options.js#L3
- https://nodejs.org/en/about/releases/

**Comment by sebdeckers** [comment]
Fix: Added an updated minimum Node.js version v10+ to `package.json` so there should be a warning at installation/runtime. Also added CI tests across all supported Node.js versions.

https://gitlab.com/commonshost/cli/-/commit/d635a0569a1c0d0af8310c439d8b47d332566ce6

**Comment by matheus-moreira** [comment]
Thank you! Now it works :)
